src/applications/vaos/components/ConfirmationDirectScheduleInfo.jsx

```diff
@@ -15,14 +15,14 @@ export default function ConfirmationDirectScheduleInfo({
   facilityDetails,
   clinic,
   pageTitle,
-  appointmentLength,
```

Comment:
We don't get an appointment length from the slots records, so I've switched to calculating it from the start and end times.